src/style.css

```diff
@@ -0,0 +1,67 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
```

Comment:
are you sure that text-align changes something here?

Reply:
yes, its not necessary here